jbrowse/src/org/labkey/jbrowse/JBrowseLuceneSearch.java

```diff
@@ -393,12 +395,30 @@ public void onUse(Query query) {
         }
     }
 
+    public static class CacheEntry {
+        private final LRUQueryCache queryCache;
+        private final IndexReader indexReader;
+
+        public CacheEntry(LRUQueryCache queryCache, IndexReader indexReader) {
```

Comment:
I dont know anything about lucene, but is caching a Reader a good idea? Does this keep file handles open?

Reply:
and if we cache them, should we be caching one reader instance per lucene index, rather than one per cache result?